scripts/release.py

```diff
@@ -433,16 +433,17 @@ def generate_commit_table(start_ref, end_ref):
     The commits in the table should only be commits that are not in the
     pull request table.
     """
-    header = "|| Author || Summary || Commit || JIRA || Verified? ||"
+    header = "|| Author || Summary || Commit || JIRA || RN? || Verified? ||"
```

Comment:
RN --> Release Notes? I think calling it RN will bring up the question of "what is it" again.